pywrap/exporter.py

```diff
@@ -314,9 +314,57 @@ def __init__(self, includes=Includes(), type_info=TypeInfo(),
         self.type_info = type_info
         self.config = config
 
-    def visit_ast(self, ast):
+    def export(self):
+        classes = self._postprocess_classes()
         self.output = render("definitions", enums=self.enums,
-                             functions=self.functions, classes=self.classes)
+                             functions=self.functions, classes=classes)
+        return super(CythonImplementationExporter, self).export()
+
+    def visit_ast(self, ast):
+        pass
+
+    def _postprocess_classes(self):
+        classes = {}
+        for class_def in self.classes:
+            classes[class_def["name"]] = class_def
+
+        # Extend class definitions with superclass
+        # TODO walk recursively, starting from leaves of inheritance tree
```

Comment:
What is a good way to represent trees in Pyhton? Hopefully without additional dependencies...